### PR TITLE
Replace API-Umbrella with Kong

### PR DESCRIPTION
**Pull request by dwendland**
New version of i4Trust tutorials incorporating Kong as default PEP/PDP

### CODE REVIEW

File: PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-kong.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+image:
+  repository: quay.io/fiware/kong
+  tag: "0.3.0-PRE-10"
```
Comment (wistefan) [inline PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-kong.yml]: We should use a stable version here, e.g. 0.2.0

File: PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-mongodb.yml [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+replicaSetHostnames: true
+
+## MongoDB containers' resource requests and limits.
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
```
Comment (wistefan) [inline PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-mongodb.yml]: Lets take out the resources. For a small test instance, this is quite some reosuces:)

File: PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-orion.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  logging:
+    level: FATAL
+  reqPoolSize: 8
+  # use this config for EntityUpdateWithTypeSubscriptionSimulation
+  notificationMode: persistent
+  # use this config for EntityUpdateWithSubscriptionSimulation
+  # second number should be around the expected clients at one instance, first is a multiple of second
+  # notificationMode: threadpool:10000:1000
+  db:
+    hosts:
```
Comment (wistefan) [inline PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-orion.yml:14]: Didn't  see that before, but do we really need 4 replicas? 2 is probably sufficient.

**Review by wistefan — CHANGES_REQUESTED** [review]

File: PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-mongodb.yml (hunk 2 above)
Comment (dwendland) [reply PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-mongodb.yml]: ok

File: PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-orion.yml (hunk 3 above)
Comment (dwendland) [reply PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-orion.yml:14]: Will reduce to standalone

File: PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-kong.yml (hunk 1 above)
Comment (dwendland) [reply PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-kong.yml]: ok

**Review by wistefan — APPROVED** [review]

**Pull request merged**